cli/install/install.go

```diff
@@ -0,0 +1,776 @@
+	}
+	err = util.CopyFilePreverse(filepath.Join(path, "tt"), filepath.Join(dst, version))
+	if err != nil {
+		printLog(logFile.Name())
+		return err
+	}
+
+	// Set symlink.
+	err = util.CreateSymLink(version, dst, "tt", true)
+	if err != nil {
+		printLog(logFile.Name())
+		return err
+	}
+	log.Infof("Done.")
+	if flags.Noclean {
+		log.Infof("Artifacts can be found at: " + path)
+	}
+	return nil
+}
+
+func checkExistingTarantool(version string, bindst string, headerDir string,
+	flags InstallationFlag) error {
+	var err error
+	if checkExisting(version, bindst) && !flags.Reinstall {
+		log.Infof("%s version of tarantool already exists, updating symlink...", version)
+		err := util.CreateSymLink(version, bindst, "tarantool", true)
+		log.Infof("Done")
+		return err
+	} else if checkExisting(version, bindst) && flags.Reinstall {
+		log.Infof("%s version of tarantool already exists, removing binary...", version)
+		os.RemoveAll(filepath.Join(bindst, version))
+	}
+
+	if checkExisting(version, headerDir) && !flags.Reinstall {
+		log.Infof("%s version of tarantool headers already exists, updating symlink...", version)
+		err := util.CreateSymLink(version, headerDir, "tarantool", true)
+		log.Infof("Done")
+		return err
+	} else if checkExisting(version, bindst) && flags.Reinstall {
+		log.Infof("%s version of tarantool binary already exists, removing headers...", version)
+		os.RemoveAll(filepath.Join(headerDir, version))
+	}
+	return err
+}
+
+// installTarantool installs selected version of tarantool.
+func installTarantool(version string, bindst string, headerDir string, flags InstallationFlag,
+	distfiles string) error {
+	tarVersion := ""
+
+	// Get latest tarantool tag if needed.
+	if version == "tarantool" {
+		tarVersion, _ = getLatestTarantoolTag()
+		version += "=" + tarVersion
+	} else {
+		tarVersion = strings.TrimPrefix(version, "tarantool=")
+	}
+	if util.IsDeprecated(tarVersion) {
+		return fmt.Errorf("Deprecated version of tarantool")
+	}
+
+	// Check bin and header dirs.
+	if bindst == "" {
+		return fmt.Errorf("Cannot procces bindir.")
+	}
+	if headerDir == "" {
+		return fmt.Errorf("Cannot procces headerdir.")
+	}
+	logFile, _ := ioutil.TempFile("", "tarantool_install")
+	defer os.Remove(logFile.Name())
+
+	log.Infof("Installing tarantool=" + tarVersion)
+
+	// Check if program is already installed.
+	log.Infof("Checking existing...")
+	err := checkExistingTarantool(version, bindst, headerDir, flags)
+	if err != nil {
+		return err
+	}
+
+	// Check dependencies.
+	log.Infof("Checking dependencies...")
+	if flags.Force {
+		log.Infof("Skipped...")
+	} else if !checkProgrammDependencies("tarantool") {
+		return nil
+	}
+
+	path, err := os.MkdirTemp("", "tarantool_install")
+	os.Chmod(path, 0777)
+	if err != nil {
+		return err
+	}
+	if !flags.Noclean {
+		defer os.RemoveAll(path)
+	}
+
+	// Download tarantool.
+	if flags.Local {
+		log.Infof("Checking local files...")
+		if checkExisting("tarantool", distfiles) {
```

Comment:
The word "tarantool" does not look like a version. Is arg name confusing or wrong argument is passed?

Reply:
Here we check that Tarantool git repo is here.